README.md

```diff
@@ -63,20 +64,16 @@ helm install nuodb-incubator/<chart>
 
 This GitHub repository contains the source for the packaged and versioned charts released in the [`gs://nuodb-charts` Google Storage bucket](https://console.cloud.google.com/storage/browser/nuodb-charts/) (the Chart Repository).
 
-The Charts in the `stable/` directory in the master branch of this repository match the latest packaged Chart in the Chart Repository, though there may be previous versions of a Chart available in that Chart Repository.
-
-The purpose of this repository is to provide a place for maintaining and contributing official Charts, with CI processes in place for managing the releasing of Charts into the Chart Repository.
-
 The Charts in this repository are organized into two folders:
 
-* stable
-* incubator
 
-Stable Charts meet the criteria in the [technical requirements](CONTRIBUTING.md#technical-requirements).
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [stable](stable/README.md)
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [incubator](incubator)
 
-Incubator Charts are those that do not meet these criteria. Having the incubator folder allows charts to be shared and improved on until they are ready to be moved into the stable folder. The charts in the `incubator/` directory can be found in the [`gs://nuodb-charts-incubator` Google Storage Bucket](https://console.cloud.google.com/storage/browser/nuodb-charts-incubator).
+The charts in the `stable` directory in the master branch of this repository are the latest packaged chart in the chart Repository. The purpose of this repository is to provide a place for maintaining and contributing official chart uupdates and enhanbcements into the chart repository.
```

Comment:
typo in `enhancements`

Reply:
capital "R" in repository mid sentence.  Also typo "uupdates"

Reply:
done. thanks for finding.